### DISCUSSION TITLE
Support for Obsidian Canvas notes

### DISCUSSION THREAD

**Discussion by CR0CKER** [alangrainger/share-note#67] [Vote for upcoming features]
It'd be great being able to upload Obsidian canvas files. Attempting to upload these notes currently produces an error.

**Comment by webenadam** [comment]
+1 i would also love to be able to share canvas notes.

**Comment by TangentFoxy** [comment]
Right now, the only workaround (while only using share note) would be to copy the JSON of a canvas into another note as a code block, and then hope that the person receiving it knows how to correctly save that content.

If anyone else knows a better method without introducing another dependency, please share!
